packages/calcite-components/src/components/table-cell/table-cell.scss

```diff
@@ -62,69 +82,61 @@ td.last-cell {
-  color: var(--calcite-color-text-1);
+.number-cell {
   background-color: var(
-    --calcite-table-cell-background-color,
-    var(--calcite-table-cell-background, var(--calcite-color-background))
+    --calcite-table-cell-number-background-color,
+    var(--calcite-table-cell-background-color, var(--calcite-color-foreground-2))
   );
+}
 
+.footer-cell {
+  @apply font-medium text-color-1;
   border-block-start: 1px solid
-    var(--calcite-table-cell-border-color, var(--calcite-table-border-color, var(--calcite-color-border-3)));
+    var(
+      --calcite-table-cell-border-color,
+      var(--calcite-table-row-border-color, var(--calcite-table-border-color, var(--calcite-color-border-3)))
+    );
+  background-color: var(--calcite-table-cell-footer-background-color, var(--calcite-color-background));
 }
 
 .number-cell,
 .selection-cell {
   @apply text-center;
   border-inline-end: 1px solid
-    var(--calcite-table-cell-border-color, var(--calcite-table-border-color, var(--calcite-color-border-3)));
+    var(
+      --calcite-table-cell-border-color,
+      var(--calcite-table-row-border-color, var(--calcite-table-border-color, var(--calcite-color-border-3)))
+    );
   inline-size: 2rem;
   min-inline-size: 2rem;
 }
 
-.number-cell {
-  color: var(--calcite-table-number-cell-text-color, var(--calcite-color-text-1));
-  background-color: var(--calcite-table-number-cell-background-color, var(--calcite-color-foreground-2));
-  &.footer-cell {
-    background-color: var(
-      --calcite-table-number-cell-background-color,
-      var(--calcite-table-cell-background-color, var(--calcite-color-background))
-    );
-  }
-}
-
 .selection-cell {
-  color: var(--calcite-table-selection-cell-icon-color, var(--calcite-color-text-3));
+  inset-inline-start: 2rem;
+  color: var(--calcite-table-cell-text-color-selected, var(--calcite-color-text-3));
+
   &:not(.footer-cell) {
     @apply cursor-pointer;
-    background-color: var(
-      --calcite-table-selection-cell-background-color,
-      var(--calcite-table-cell-background-color, transparent)
-    );
-  }
-  &.footer-cell {
-    background-color: var(
-      --calcite-table-selection-cell-background-color,
-      var(--calcite-table-cell-background-color, var(--calcite-color-background))
-    );
   }
 }
 
 .selected-cell:not(.number-cell):not(.footer-cell) {
-  background-color: var(
+  --calcite-internal-table-cell-background-color: var(
     --calcite-table-cell-background-color-selected,
-    var(--calcite-table-row-background-color-selected, var(--calcite-color-foreground-current))
+    var(--calcite-color-foreground-current)
   );
 }
 
 .selection-cell.selected-cell {
-  box-shadow: inset 0.25rem 0 0 0 var(--calcite-table-row-selected-accent-color, var(--calcite-color-brand));
-  color: var(--calcite-table-selection-cell-icon-color-selected, var(--calcite-color-brand));
+  box-shadow: var(--calcite-table-cell-shadow-selected, inset 0.25rem 0 0 0 var(--calcite-color-brand));
```

Comment:
I wonder if the accent terminology is more understandable to an end user. It’s really creating a border accent - like we have in alert, notice, etc. this terminology with shadow makes it seem like the shadow is customizable.

Reply:
It’s also only applicable to the single selection cell that receives the accent so I think the prior naming makes more sense. It should only be set on the row once, and not on an individual component. The cell that receives this color override is rendered internally within the Table Row.

Reply:
thanks for the catch! But I think naming wise it aligns more with the "indicator" pattern that exists in other components.

Reply:
Yeah, I know Alert uses "accent" for a similar thing, I think we have "highlight" in use (or talked about it). I think as long as we don't use shadow here I'm ok with it. 

Really in plain terms this is "the color of the icon in the selectable checkbox cell when that cell is selected". Because the user can't target this Cell directly, I think using terminology that has Table Row makes sense, since it can be controlled once at the app level or still individually per row. 

"--calcite-table-row-indicator-selected-color", "--calcite-table-row-selected-highlight/indicator", etc., would maybe solve both our concerns?